### ISSUE TITLE
bug: InjectedParamAnnotationOrder missing import guard for parameter-level InjectedParam

### ISSUE THREAD

**Issue by kirich1409** [androidbroadcast/Koin-Detekt#103]
### Problem
`annotations/InjectedParamAnnotationOrder.kt` line 55:
```kotlin
val injectedParamIndex = annotations.indexOf("InjectedParam")
```
The class-level annotation uses `importContext` but the parameter-level lookup doesn't. Any non-Koin `@InjectedParam` from another library could produce a false positive.

### Fix
Verify `InjectedParam` resolves to Koin via `importContext.resolveKoin("InjectedParam")`.
